apps/keyboard/test/marionette/lib/keyboard.js

```diff
@@ -56,11 +61,36 @@ Keyboard.prototype = {
     return this.client.findElement(Keyboard.Selector.shiftKey);
   },
 
+  get backspaceKey() {
+    return this.client.findElement(Keyboard.Selector.backspaceKey);
+  },
+
+  get activeKeyboardFrame() {
+    return this.client.findElement(Keyboard.Selector.activeKeyboardFrame);
+  },
+
+  get autoCorrect() {
+    return this.client.settings.get('keyboard.autocorrect');
+  },
+
+  get predictiveWord() {
+    return this.client.findElement(Keyboard.Selector.predictiveWord);
+  },
+
+  set autoCorrect(val) {
+    this.client.settings.set('settings-write', true);
+    this.client.settings.set('settings-api-write', true);
+    this.client.settings.set('keyboard.autocorrect', val);
```

Comment:
Let's remove this API and actually go to the Keyboard Settings panel and toggle these switches (in the tests that uses this API).